frontend/src/components/activity/CardContentNode.vue

```diff
@@ -85,6 +108,19 @@ export default {
     icon() {
       return this.$tc(`contentNode.${camelCase(this.contentNode.contentTypeName)}.icon`)
     },
+    infoRows() {
```

Comment:
I understand the this solves the problem, still looks very quirky to me 😃 

Any other options you considered?
- Putting line breaks directly into the translation file? https://github.com/kazupon/vue-i18n/issues/18
- Using i18n tag to interpolate breaks? https://stackoverflow.com/a/65862151
- Using css `white-space: pre-line` https://stackoverflow.com/a/68813580

Reply:
used this idea:  https://stackoverflow.com/a/70755625